src/modules/recruitments/services/recruitments.service.ts

```diff
@@ -0,0 +1,151 @@
+import { PrismaClient } from '@prisma/client';
+import {
+  CreateRecruitmentProps,
+  GetListRecruitmentParams,
+  UpdateRecruitmentProps,
+} from '../type';
+import { getPageAndLimitFromParams } from '../../../utils';
+
+const isRecruitmentExisted = async (
+  tenantPrisma: PrismaClient,
+  recruitmentId: string,
+) => {
+  const recruitment = await tenantPrisma.recruitment.findFirst({
+    where: {
+      AND: [{ id: recruitmentId }, { archived: false }],
+    },
+  });
+  if (!recruitment) {
+    throw new Error('Recruitment is not Existed');
+  }
+};
+
+export default class RecruimentService {
+  static create = async (
+    tenantPrisma: PrismaClient,
+    recruitmentOwnerId: string,
+    body: CreateRecruitmentProps,
+  ) => {
+    const newRecruitment = await tenantPrisma.recruitment.create({
+      data: {
+        ...body,
+        recruitmentOwnerId: recruitmentOwnerId,
+      },
+    });
+    await tenantPrisma.$disconnect();
```

Comment:
why?

Reply:
Testing something and I forgot to delete, sorry.